compound/src/main/kotlin/io/element/android/compound/previews/ColorPreview.kt

```diff
@@ -54,6 +60,7 @@ fun ColorPreview(
                 Box(
                     modifier = Modifier
                         .padding(1.dp)
+                        .background(Color.White)
```

Comment:
Yes: without this, the colors that have an alpha value are overlayed directly above this component's parent background, which is a gradient, and it's difficult to really understand what the color looks like unless you take a look at both extremes of each item (the pure white/black colors).

Although it might be that what we wanted when using the gradient, I'm not sure. I found it quite confusing myself.

Reply:
The gradient idea was to add a border with "several" colors. So it's fine to set a White bg, I did not think about colors with alpha. Thanks!